src/overlays/actors/ovl_Npc_Engineer/ac_npc_engineer.c

```diff
@@ -20,19 +22,51 @@ ActorProfile Npc_Engineer_Profile = {
     /* */ aNEG_actor_ct,
     /* */ aNEG_actor_dt,
     /* */ aNEG_actor_init,
-    /* */ (ActorFunc)none_proc1,
+    /* */ (void*)none_proc1,
     /* */ aNEG_actor_save,
 };
-#endif
 
-#pragma GLOBAL_ASM("asm/jp/nonmatchings/overlays/actors/ovl_Npc_Engineer/ac_npc_engineer/aNEG_actor_ct.s")
+struct_809AEFA4 D_809AEFA4_jp = {
+    func_809AEF28_jp,
+    func_809AEF54_jp,
+    4,
+    0,
+};
+
+void aNEG_actor_ct(Actor* thisx, Game_Play* game_play) {
+    Npc_Engineer* this = (Npc_Engineer*)thisx;
+    if (common_data.unk_1004C->unk_BC(thisx, game_play) == 1) {
```

Comment:
Could you leave an empty line between the declaration and the rest of the code?
```suggestion
    Npc_Engineer* this = (Npc_Engineer*)thisx;

    if (common_data.unk_1004C->unk_BC(thisx, game_play) == 1) {
```